java/format/pom.xml

```diff
@@ -15,7 +15,7 @@
 <parent>
   <artifactId>arrow-java-root</artifactId>
   <groupId>org.apache.arrow</groupId>
-  <version>4.0.0</version>
+  <version>siren-4.0.0-1-SNAPSHOT</version>
```

Comment:
need stable version later on

Reply:
you'll make the release from `branch-siren-4.0.x` ?